src/seedu/addressbook/data/person/Person.java

```diff
@@ -9,13 +9,18 @@
  * Guarantees: details are present and not null, field values are validated.
  */
 public class Person implements ReadOnlyPerson {
-
+    
+    public static int nextSequenceNumber = 1;
```

Comment:
this doesn't have to be public. in general, try to make all members private first, then protected, use public as last resort